python/tvm/contrib/debugger/debug_executor.py

```diff
@@ -268,14 +268,18 @@ def run_individual(self, number, repeat=1, min_repeat_ms=0):
         ret = self._run_individual(number, repeat, min_repeat_ms)
         return ret.strip(",").split(",") if ret else []
 
-    def profile(self, **input_dict):
+    def profile(self, collectors=[], **input_dict):  # pylint: disable=dangerous-default-value
```

Comment:
I see you have a `# pylint: disable=dangerous-default-value`, after setting the `collectors=[]`.

Just wanted to double check whether this is intended, as `collectors=[]` behaves like a global, and there will be only one list that will be shared across all the calls to `profile()` in which collectors is not provided?

Reply:
This list is never modified, so it should be fine.